tasks/setup_cron.yml

```diff
@@ -0,0 +1,38 @@
+- name: CRON Peering-DB Sync
+  cron:
+    name: "Sync Peering-DB"
+    minute: "30"
+    hour: "2"
+    job: "cd {{ peering_manager_install_directory }} && python3 manage.py peeringdb_sync"
```

Comment:
Cron jobs must use the Python that is installed inside the virtualenv which is installed.